djangocms_article_drafts/models.py

```diff
@@ -41,10 +109,11 @@ def __contains__(self, model_class):
 
     def create_publishable(self, sender, instance, created):
         if created:
-            Publishable.objects.create(draft_object=instance)
+            Publishable.create(draft_object=instance)
```

Comment:
Not sure what the point of this classmethod is, if there is no special behaviour distinct from the `objects` default manager. Can keep it for now I suppose, may do custom logic.

Reply:
Yup, that was the idea. We could check if it exists already, etc